presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeExecution.java

```diff
@@ -41,23 +45,34 @@
     protected QueryRunner createQueryRunner()
             throws Exception
     {
-        return PrestoSparkQueryRunner.createHivePrestoSparkQueryRunner();
+        Map<String, String> configs = new HashMap<>();
+        // prevent to use the default Prestissimo config files since the Presto-Spark will generate the configs on-the-fly.
+        configs.put("catalog.config-dir", "/");
+        return PrestoSparkQueryRunner.createHivePrestoSparkQueryRunner(configs);
     }
 
     @Test
-    public void testNativeExecutionWithTableWrite()
+    public void testNativeExecutionWithProjection()
     {
+        assertUpdate("CREATE TABLE test_order WITH (format = 'DWRF') as SELECT orderkey, custkey FROM orders LIMIT 100", 100);
+
+        String prestoServerPath = System.getProperty("PRESTO_SERVER");
+        assertNotNull(prestoServerPath);
+
         Session session = Session.builder(getSession())
                 .setSystemProperty(NATIVE_EXECUTION_ENABLED, "true")
+                .setSystemProperty(NATIVE_EXECUTION_EXECUTABLE_PATH, prestoServerPath)
                 .setSystemProperty("table_writer_merge_operator_enabled", "false")
                 .setCatalogSessionProperty("hive", "collect_column_statistics_on_write", "false")
+                .setCatalogSessionProperty("hive", PUSHDOWN_FILTER_ENABLED, "true")
                 .build();
 
-        // Expecting 0 row updated since currently the NativeExecutionOperator is dummy.
-        assertUpdate(session, "CREATE TABLE test_tablescan as SELECT orderkey, custkey FROM orders", 0);
```

Comment:
Curious, how did the previous test pass when operator is not yet implemented?

Reply:
Previously the `NativeExecutionOperator` is dummy which returns a empty result as output, so in the test case here we expect zero row returns from the query.

Reply:
I see. So the CREATE query did not actually run?

Reply:
It runs, but will not actually create the table as the SELECT query return empty result.

Reply:
Ah, right